site/en/tutorials/structured_data_extraction.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<p>Nit: You don't need to capitalize "function calling".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>